drivers/storage/portworx/util/util.go

```diff
@@ -1172,3 +1172,8 @@ func CountStorageNodes(
 
 	return storageNodesCount, nil
 }
+
+// IsFreshInstall checks whether it's a fresh Portworx install
+func IsFreshInstall(cluster *corev1.StorageCluster) bool {
+	return cluster.Status.Phase == ""
```

Comment:
Would this cause any issues on retries? Is the phase updated if it fails the first time?

Reply:
good catch, after review I saw one corner case here, but not in scope of this ticket. I'm going to create a bug for that.
basically even if it's a fresh install, before stc sync during validation if it fails, phase will be set to Failed, then next retry it won't be able to set this default number of storage node. So far I don't have any easy solutions, but should be able to fix this once finish this https://portworx.atlassian.net/browse/PWX-27765. cc: @piyush-nimbalkar